src/python/build_management/build_manager.py

```diff
@@ -1153,7 +1153,20 @@ def setup_trunk_build():
     logs.log_error('Unable to find a matching revision.')
     return None
 
-  return setup_regular_build(revision)
+  build = setup_regular_build(revision)
+  if build:
+    setup_auxiliary_builds(revision)
+
+  return build
+
+
+def setup_auxiliary_builds(revision):
```

Comment:
since this is specific to fuzzing, it seems like this particular function belongs in fuzz_task.

Reply:
Yes can go there.

Reply:
Not sure I understand what you mean. I agree that it's better to call this function from the fuzz_task, as auxiliary builds aren't needed for any other task. But I think the function itself should stay in the build_manager, as it would need to use some of the classes / functions defined in the build_manager.

Reply:
setup_auxiliary_builds() should belong in fuzz_task, since it will only ever be called in fuzzing. We can still call classes / functions defined in build_manager from there.

Reply:
so, inside fuzz_task:

```
def set_up_dataflow_build(...):
  build_manager.RegularBuild()

build_manager.setup_build(...)
set_up_data_flow_build(...)
```

You may need to do some minor refactoring to avoid the APP_PATH/APP_DIR etc from being overwritten.

Reply:
Alright, done! Thanks for mentioning the env variables, I think I've made sure those won't be overwritten, but I'll double check it in tests.